build.gradle

```diff
@@ -81,14 +81,14 @@ configurations.all {
         force 'commons-cli:commons-cli:1.3.1'
         force 'org.apache.httpcomponents:httpcore:4.4.12'
         force "org.apache.commons:commons-lang3:3.4"
-        force "org.springframework:spring-core:5.3.28"
-        force "org.springframework:spring-expression:5.3.28"
+        force "org.springframework:spring-core:5.3.39"
```

Comment:
Also force spring-context to 5.3.39 to resolve the whitesource error:

```
force "org.springframework:spring-context:5.3.39"
```

https://github.com/opensearch-project/security/pull/4947/checks?check_run_id=33937343697

Reply:
done